src/DynamoCoreWpf/ViewModels/Menu/PreferencesViewModel.cs

```diff
@@ -199,6 +199,19 @@ public string SelectedNumberFormat
             }
         }
 
+        public double SelectedDefaultScaleFactor
```

Comment:
comments.

Reply:
Comments were added in the next PR: https://github.com/DynamoDS/Dynamo/pull/13734